gradle-jdks-setup/src/main/resources/gradle-jdks-setup.sh

```diff
@@ -129,36 +125,55 @@ for dir in "$APP_GRADLE_DIR"/jdks/*/; do
   # Check if distribution exists in $GRADLE_JDKS_HOME
   jdk_installation_directory="$GRADLE_JDKS_HOME"/"$distribution_local_path"
   if [ -d "$jdk_installation_directory" ]; then
-    echo "Distribution $distribution_url already exists in $jdk_installation_directory"
+    echo "Distribution '$distribution_url' already exists in '$jdk_installation_directory'"
```

Comment:
I think we'll want to hide any output in the case where everything is already set up to avoid noise - at least in the `./gradlew` case (IntelliJ will go to it's own tool window so less of a problem).